src/cppcache/integration-test/testThinClientRemoteQuerySS.cpp

```diff
@@ -56,59 +57,63 @@ const wchar_t* checkNullString(const wchar_t* str) {
   return ((str == nullptr) ? L"(null)" : str);
 }
 
+std::string checkNullString(const std::string* str) {
+    return ((str == nullptr) ? "(null)" : *str);
+}
+
 void _printFields(CacheablePtr field, Struct* ssptr, int32_t& fields) {
   if (auto portfolio = std::dynamic_pointer_cast<Portfolio>(field)) {
     printf("   pulled %s :- ID %d, pkid %s\n",
-           checkNullString(ssptr->getFieldName(fields)), portfolio->getID(),
+           checkNullString(ssptr->getFieldName(fields)).c_str(), portfolio->getID(),
```

Comment:
I think I would just rather see the tests not log so much crap and if they do that they use a logger with stream functionality. 
```
if (log.debug) {
  log.debug << "Doing some logging. somevar=" << someVar << ", someOtherVar=" << someOtherVar;
}
```
😒